### DISCUSSION TITLE
allow decrease in vertical tab spacing -FF 89 has added useless space between tabs in TST

### DISCUSSION THREAD

**Discussion by DrDrTed** [piroor/treestyletab#2919] [Ideas]
Once again, the "modern" design (usually mobile based) increases useless empty space in TST, decreasing the number of tabs that can be shown on a screen significantly. (by causing the vertical spacing between tabs to increase)

My request: allow users to decrease the vertical spacing between tabs. If that is very difficult, then changing the spacing back to the previous spacing (by over-writing something FF is doing in ver 89) would be a great improvement.

If there is already an option or an add-on to change the vertical spacing between tabs, please accept my apologies. If not, I think that, for basic usability, restoring the number of tabs visible is very basic and worth working on.

**Comment by piroor** [comment]
Did you try this?
https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#change-tab-height-236-2389

**Reply by Adelscott** [reply]
Thanks  a lot !
